config/schedule.yml

```diff
@@ -1,113 +1,139 @@
   cron: "10 */12 * * *" # 10 min past every 12th hour UTC
   class: "BadgeAchievements::BadgeAwardWorker"
   args:
     - ""
     - award_four_week_streak
     - ""
 award_eight_week_streak_badge:
+  description: "Awards 'eigth week publishing streak' badges to users (runs every 12 hours on the 15th minute after the hour)"
   cron: "15 */12 * * *" # 15 min past every 12th hour UTC
   class: "BadgeAchievements::BadgeAwardWorker"
   args:
     - ""
     - award_eight_week_streak
     - ""
 award_sixteen_week_streak_badge:
+  description: "Awards 'sixteen week publishing streak' badges to users (runs every 12 hours on the 20th minute after the hour)"
   cron: "20 */12 * * *" # 20 min past every 12th hour UTC
   class: "BadgeAchievements::BadgeAwardWorker"
   args:
     - ""
     - award_sixteen_week_streak
     - ""
 award_weekly_tag_badges:
+  description: "Awards 'weekly beloved article for a tag' badges to users (runs at 11:00 UTC on Thursday)"
   cron: "0 11 * * 4" # 11 am UTC every Thursday
   class: "BadgeAchievements::BadgeAwardWorker"
   args:
     - ""
     - award_tag
     - ""
 award_contributor_badges_from_github:
+  description: "Awards 'GitHub contributor' badges to users (runs hourly on the 20th minute after the hour)"
   cron: "20 * * * *" # every hour, 20 min after the hour
   class: "BadgeAchievements::BadgeAwardWorker"
   args:
     - ""
     - award_contributor_from_github
     - ""
 remove_old_html_variant_data:
+  description: "Deletes old HTML variants (runs hourly on the 10th minute after the hour)"
   cron: "10 * * * *" # every hour, 10 min after the hour
   class: "HtmlVariants::RemoveOldDataWorker"
 resave_supported_tags:
+  description: "Resaves supported tags to recalculate scores (runs daily at 00:25 UTC)"
   cron: "25 0 * * *" # daily at 12:25 am UTC
   class: "Tags::ResaveSupportedTagsWorker"
 expire_old_listings:
+  description: "Expires old listings (runs daily at 00:30 UTC)"
   cron: "30 0 * * *" # daily at 12:30 am UTC
   class: "Listings::ExpireOldListingsWorker"
 send_welcome_notifications:
+  description: "Sends welcome notifications to new users (runs daily at 16:30 UTC)"
   cron: "0 16 * * *" # daily at 4 pm UTC
   class: "Broadcasts::SendWelcomeNotificationsWorker"
 sitemap_refresh:
+  description: "Regenerates the sitemap and pings search engines (runs hourly on the 30th minute after the hour)"
   cron: "30 * * * *" # every hour, 30 min after the hour
   class: "SitemapRefreshWorker"
 hourly_feed_cache_bust:
+  description: "Busts the edge cache of the feed (runs hourly on the hour)"
   cron: "0 * * * *" # hourly on the hour
   class: "BustCachePathWorker"
   args:
     - "/feed.xml"
 hourly_badge_cache_bust:
+  description: "Busts the edge cache of badges (runs hourly on the hour)"
   cron: "0 * * * *" # hourly on the hour
   class: "BustCachePathWorker"
   args:
     - "/badge"
 daily_home_cache_bust:
+  description: "Busts the edge cache of the homepage (runs hourly on the hour)"
```

Comment:
Base on the comment below, should be `runs hourly on the hour`  or `runs daily at 12am UTC`?

Reply:
You're right, it runs only at midnight UTC. Let me update that, too much copying and pasting :D